cfy_manager/main.py

```diff
@@ -535,18 +539,34 @@ def _print_finish_message():
                 protocol=protocol,
                 ip=manager_config[PUBLIC_IP])
         )
-        print_password_to_screen()
+        print_credentials_to_screen()
         logger.notice('#' * 50)
         logger.notice("To install the default plugins bundle run:")
         logger.notice("'cfy plugins bundle-upload'")
         logger.notice('#' * 50)
 
 
-def print_password_to_screen():
+def print_credentials_to_screen():
     password = config[MANAGER][SECURITY][ADMIN_PASSWORD]
+    manager_status_reporter_token = config.get(
+        MANAGER_STATUS_REPORTER, {}).get(TOKEN, None)
+    db_status_reporter_token = config.get(
+        DB_STATUS_REPORTER, {}).get(TOKEN, None)
+    queue_status_reporter_token = config.get(
+        QUEUE_STATUS_REPORTER, {}).get(TOKEN, None)
+
     current_level = get_file_handlers_level()
     set_file_handlers_level(logging.ERROR)
     logger.warning('Admin password: {0}'.format(password))
+    if manager_status_reporter_token:
+        logger.warning('Manager Status Reported token: {0}'
+                       ''.format(manager_status_reporter_token))
```

Comment:
i say for new logs let's use the correct form, ie. `logger.warning('Manager Status Reported token: %s', manager_status_reporter_token)`